libaftermath-core/src/defs/aftermath/types/openmp/on_disk.py

```diff
@@ -248,7 +248,7 @@
             field_type = aftermath.types.builtin.uint64_t,
             comment = "Time of the event")]))
 
-################################################################################
+#################################################################################
```

Comment:
This was probably not supposed to be in the commit

Reply:
Resolved when commits get squashed.